closure/goog/testing/asserts_test.js

```diff
@@ -436,9 +438,61 @@ function testAssertObjectEqualsMap() {
   assertObjectEquals('maps should be equal', map1, map2);
 
   map1.set('hi', 'hey');
-  assertThrows('sets should not be equal', function() {
-    assertObjectEquals(map1, map2);
-  });
+  assertThrows('maps should not be equal',
+               function() { assertObjectEquals(map1, map2); });
+}
+
+
+function testAssertObjectEqualsTypedArray() {
+  var typedArrays = [
+    'Float32Array',
+    'Float64Array',
+    'Int8Array',
+    'Int16Array',
+    'Int32Array',
+    'Uint8Array',
+    'Uint8ClampedArray',
+    'Uint16Array',
+    'Uint32Array',
+    'BigInt64Array',
+    'BigUint64Array',
+  ];
+
+  var prior, priorName;
```

Comment:
May be worth a comment that we're saving the prior in order to check that the same array of different types is not equal?